### PR TITLE
create contact html

### PR DESCRIPTION
**Pull request by HebaAhmad1**
relates #23

### CODE REVIEW

File: contact/contact.html [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/620f3808e1.js" crossorigin="anonymous"></script>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./contact.css">
+    <title>Document</title>
```
Comment (Ibrahim-Jarada) [inline contact/contact.html:12]: change the title to Contact

File: contact/contact.html [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
+</head>
+<body>
+    <header class="header">
+        <!-- <img src="../assets/Vector.png" class="back"> -->
```
Comment (Ibrahim-Jarada) [inline contact/contact.html:16]: why you have code line comment? if it is outdated just delete it

File: contact/contact.html [hunk 3]
```diff
@@ -1,11 +1,29 @@
...
+        <i class="fas fa-arrow-left back" ></i>   
+       <h1 class="covid-19">C<img src="../assets/coviddd.png" class="covid-img">vid-19</h1>
+        <img src="../assets/dark-mode.png" class="dark-mode">
+    </header>
+    <section class="container">
+        <section class="part ">
```
Comment (Ibrahim-Jarada) [inline contact/contact.html:22]: Why did you use one section element inside another?
Take a look of this code example
```<header></header>
<section>
	<article>
		<figure>
			<img>
			<figcaption></figcaption>
		</figure>
	</article>
</section>
```

**Review by Ibrahim-Jarada — COMMENTED** [review]
Use Prettier - Code formatter  in the last commit before you push the branch
Keyboard Shortcut => Format Document : Shift+Alt+F

**Review by NadaSaleh20 — DISMISSED** [review]

**Review by AbdallahGot — APPROVED** [review]

**Review by NadaSaleh20 — DISMISSED** [review]

**Review by Ibrahim-Jarada — APPROVED** [review]

**Review by Rawanzaqout — APPROVED** [review]

**Review by NadaSaleh20 — APPROVED** [review]

**Review by AbdallahGot — APPROVED** [review]

**Pull request merged**